sim/envs/humanoids/stompymicro_config.py

```diff
@@ -0,0 +1,244 @@
+
+        default_positions = Robot.default_standing()
+        for joint in default_positions:
+            default_joint_angles[joint] = default_positions[joint]
+
+    class control(LeggedRobotCfg.control):
+        # PD Drive parameters:
+        stiffness = Robot.stiffness()
+        damping = Robot.damping()
+        # action scale: target angle = actionScale * action + defaultAngle
+        action_scale = 0.25
+        # decimation: Number of control action updates @ sim DT per policy DT
+        decimation = 10  # 100hz
+
+    class sim(LeggedRobotCfg.sim):
+        dt = 0.001  # 1000 Hz
+        substeps = 1  # 2
+        up_axis = 1  # 0 is y, 1 is z
+
+        class physx(LeggedRobotCfg.sim.physx):
+            num_threads = 10
+            solver_type = 1  # 0: pgs, 1: tgs
+            num_position_iterations = 4
+            num_velocity_iterations = 1
+            contact_offset = 0.01  # [m]
+            rest_offset = 0.0  # [m]
+            bounce_threshold_velocity = 0.1  # [m/s]
+            max_depenetration_velocity = 1.0
+            max_gpu_contact_pairs = 2**23  # 2**24 -> needed for 8000 envs and more
+            default_buffer_size_multiplier = 5
+            # 0: never, 1: last sub-step, 2: all sub-steps (default=2)
+            contact_collection = 2
+
+    class domain_rand(LeggedRobotCfg.domain_rand):
+        start_pos_noise = 0.1
+        randomize_friction = True
+        friction_range = [0.1, 2.0]
+
+        randomize_base_mass = True
+        added_mass_range = [-1.0, 1.0]
+        push_robots = True
+        push_interval_s = 4
+        max_push_vel_xy = 0.2
+        max_push_ang_vel = 0.4
+        dynamic_randomization = 0.05
+
+    class commands(LeggedRobotCfg.commands):
+        # Vers: lin_vel_x, lin_vel_y, ang_vel_yaw, heading (in heading mode ang_vel_yaw is recomputed from heading error)
+        num_commands = 4
+        resampling_time = 8.0  # time before command are changed[s]
+        heading_command = True  # if true: compute ang vel command from heading error
+
+        class ranges:
+            lin_vel_x = [-0.3, 0.6]  # min max [m/s]
+            lin_vel_y = [-0.3, 0.3]  # min max [m/s]
+            ang_vel_yaw = [-0.3, 0.3]  # min max [rad/s]
+            heading = [-3.14, 3.14]
+
+    class rewards:
+        base_height_target = 0.78
+        min_dist = 0.25
+        max_dist = 0.5
+
+        # put some settings here for LLM parameter tuning
+        target_joint_pos_scale = 0.17  # rad
+        target_feet_height = 0.05  # m
+        cycle_time = 0.4  # sec
+        # if true negative total rewards are clipped at zero (avoids early termination problems)
+        only_positive_rewards = True
+        # tracking reward = exp(error*sigma)
+        tracking_sigma = 5.0
+        max_contact_force = 400  # forces above this value are penalized
+
+        class scales:
+            # reference motion tracking
+            joint_pos = 1.6
```

Comment:
```suggestion
            joint_pos = 1.6
                        feet_clearance = 1.6
            feet_contact_number = 1.2
            # gait
            feet_air_time = 1.6
            foot_slip = -0.05
            feet_distance = 0.2
            knee_distance = 0.2
            # contact
            feet_contact_forces = -0.01
            # vel tracking
            tracking_lin_vel = 1.2
            tracking_ang_vel = 1.1
            vel_mismatch_exp = 0.5  # lin_z; ang x,y
            low_speed = 0.2
            track_vel_hard = 0.5
```

for standing - comment this part ou